app-cdr/daa2iso/daa2iso-0.1.7e-r1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+inherit toolchain-funcs
+
+DESCRIPTION="Convert CD images from daa/gbi to iso"
+HOMEPAGE="http://aluigi.altervista.org/mytoolz.htm"
+SRC_URI="http://aluigi.altervista.org/mytoolz/${PN}.zip -> ${P}.zip"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux ~x64-macos"
+
+DEPEND="app-arch/unzip"
+
+S="${WORKDIR}"
+
+src_compile() {
+	emake CC="$(tc-getCC)" -C src -f - <<-'EOF'
+	      daa2iso: $(patsubst %.c,%.o,$(wildcard *.c))
```

Comment:
Did you test it? I think heredoc expands bash vars in place, i.e. you need to escape `$()`.